addons/udes_hierarchical_tree_view/.gitignore

```diff
@@ -0,0 +1,128 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Should this be here? I haven't checked elsewhere but usually `.gitignore` is only at the root of the repo.